scipy/fft/_pocketfft/pypocketfft.cxx

```diff
@@ -174,22 +188,23 @@ template<typename T> py::array irfftn_internal(const py::array &in,
   auto s_out=copy_strides(res);
   auto d_in=reinterpret_cast<const complex<T> *>(in.data());
   auto d_out=reinterpret_cast<T *>(res.mutable_data());
+  auto fct = norm_fct<T>(norm, dims_out, axes);
   {
   py::gil_scoped_release release;
-  c2r(dims_out, s_in, s_out, axes, d_in, d_out, T(fct), nthreads);
+  c2r(dims_out, s_in, s_out, axes, d_in, d_out, fct, nthreads);
```

Comment:
can't the call to `norm_fct` also go within the section with the GIL released? (the same pattern with `norm_fct` occurs elsewhere as well)

Reply:
I've moved them all to a scoped release region.